src/xdocs/config_system_properties.xml

```diff
@@ -16,15 +16,6 @@
       </macro>
     </section>
 
-    <section name="Localization Support">
-      <p>
-        Checkstyle supports localization of the output messages.
-        If your language is not supported, please consider
-        translating the messages in the <code>messages.properties</code> file. Please let us
-        know if you translate the file.
-      </p>
-    </section>
-
```

Comment:
Why you removed this? We support localization right?

Reply:
it is unrelated to system properties, page is for system properties